cfpb/consumerfinance.gov
# Retirement spanish urls

Depends on #8424

Switches the retirement app to the `/es/` prefix format.

Also adds the standard language picker to the about page for the retirement app

## Testing
- Load up http://localhost:8000/consumer-tools/retirement/before-you-claim/
- Run through the tool, everything should function as normal.
- Click through to the about page via the [Where do these numbers come from?](http://localhost:8000/consumer-tools/retirement/before-you-claim/about/)
- This should work (and the page should have a language picker
- Give the language picker a try
- Switch to Español back in the main app
- Everything should work.
